scripts/ec2cleanup.py

```diff
@@ -75,23 +79,76 @@ def delete_unattached_volumes(provider_mgmt, excluded_volumes, output):
         logger.error(e)
 
 
-def ec2cleanup(exclude_volumes, exclude_eips, output):
+def delete_unused_loadbalancers(provider_mgmt, excluded_elbs, output):
+    elb_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for elb in provider_mgmt.get_all_unused_loadbalancers():
+            if excluded_elbs and elb.name in excluded_elbs:
+                print "  Excluding Elastic LoadBalancer id: {}".format(elb.name)  # noqa
+                continue
+            else:
+                elb_list.append([provider_name, elb.name])
+                provider_mgmt.delete_loadbalancer(loadbalancer=elb)
+        print "  Deleted Elastic LoadBalancers: {}".format(elb_list)  # noqa
+        with open(output, 'a+') as report:
+            if elb_list:
+                # tabulate volume_list and write it
+                report.write(tabulate(tabular_data=elb_list,
+                                      headers=['Provider Key', 'ELB name'],
+                                      tablefmt='orgtbl'))
+    except Exception as e:
+        logger.error(e)
```

Comment:
**Required**
Let's print something here as well as using logger, so its apparent in the job output that there was an issue. Otherwise its a 'silent' failure as the job output and the script log (not the same as logger here) don't have any failure messages.

I'm going to create an issue to actually remove the logger use in this, and instead print everything so that its quickly and easily accessible when reviewing the job in Jenkins.